grammar.js

```diff
@@ -523,19 +527,33 @@ module.exports = grammar({
       seq(
         '######"',
         repeat(choice(
-          $._sl6_string_chars,
+          alias($.slStringLiteralPart6, $.slStringLiteralPart),
           alias($.escapeSequence6, $.escapeSequence),
           alias($.interpolationExpr6, $.interpolationExpr)
         )),
         '"######'
       ),
     ),
 
+    slStringLiteralPart: $ => token.immediate(/[^"\\\n\r]+/),
```

Comment:
External rules are not generated as children in their parents, so I need to have normal rules here wrapping the external ones.